research/src/relativistic_hmc.jl

```diff
@@ -1,3 +1,5 @@
+using AdvancedHMC
```

Comment:
The CI is breaking as we will need to `using` internal names within `AdvancedHMC` inside here, such as `using AdvancedHMC: AbstractKinetic`. And potentially for all other names that we do not export but actually need to use or extend.